docs/docsite/rst/os_guide/intro_zos.rst

```diff
@@ -0,0 +1,247 @@
+and automated in an Ansible playbook.
+
+
+The z/OS landscape
+------------------
+While most systems process files in two modes - binary or text encoded in UTF-8,
+IBM z/OS including UNIX System Services features an additional third mode - text encoded in EBCDIC.
+Ansible has provisions to handle binary data and UTF-8 encoded textual data, but not EBCDIC encoded data.
+This is not necessarily a limitation, it simply requires additional tasks that convert files to and from their original encodings.
+It is up to the Ansible user managing z/OS UNIX nodes to understand the nature of the files in their automation.
+
+The type (binary or text) and encoding of files can be stored in file "tags".
+File tags is a z/OS UNIX System Services concept (part of Enhanced ASCII) designed to distinguish binary
+files from UTF-8 encoded text files and EBCDIC-encoded text files.
+
+Default behavior for an un-tagged file or stream is determined by the program, for example,
+`IBM Open Enterprise SDK for Python <https://www.ibm.com/products/open-enterprise-python-zos>`__ defaults to the UTF-8 encoding.
+
+Ansible modules will not read or recognize file tags. It is up to the user to determine the nature of remote data and tag it appropriately.
+Data sent to remote z/OS UNIX hosts through Ansible is, by default, encoded in UTF-8 and not tagged.
+Tagging a file is achievable with an additional task using the ``ansible.builtin.command`` module.
+
+.. code-block:: yaml
+
+    - name: Tag my_file.txt as UTF-8.
+      ansible.builtin.command: chtag -tc iso8859-1 my_file.txt
+
+
+The `z/OS shell <https://www.ibm.com/docs/en/zos/latest?topic=shells-introduction-zos>`_ available on
+z/OS UNIX System Services defaults to an EBCDIC encoding for un-tagged data streams.
+Ansible sends untagged UTF-8 encoded textual data to the z/OS shell which expects untagged data to be encoded in EBCDIC.
+This mismatch in data encodings can be resolved by setting the ``PYTHONSTDINENCODING`` environment variable,
+which causes the pipe used by Python to be tagged with the specified encoding.
+File and pipe tags can be used with automatic conversion between ASCII and EBCDIC, but only programs on
+z/OS UNIX which are aware of tags will use them.
+
+
+Using ``ansible.builtin`` modules with z/OS UNIX
+------------------------------------------------
+
+The ``ansible.builtin`` modules operate under the assumption that all textual data (files and pipes/streams) is UTF-8 encoded.
+On z/OS, since textual data (file or stream) is sometimes encoded in EBCDIC and sometimes in UTF-8, special care must be taken to identify the correct encoding of target data.
+
+Here are some notes / pro-tips when using the ``ansible.builtin`` modules with z/OS UNIX. This is by no means a comprehensive list.
+Before using any Ansible modules, you must first :ref:`configure_zos_remote_environment`.
+
+* ansible.builtin.command / ansible.builtin.shell
```

Comment:
I'd link to the modules in question:
```suggestion
* :ansplugin:`ansible.builtin.command#module` / :ansplugin:`ansible.builtin.shell#module`
```
Then users can quickly navigate to the module docs by clicking on the names, instead of having to search for them.